scss/modifierClasses/_sticky.scss

```diff
@@ -0,0 +1,10 @@
+.sticky {
```

Comment:
should we doc this or are these private for some reason?

If they're meant for public consumption, use the hologram-style `/*doc` comment blocks

Reply:
That was just an oversight on my part. Adding docs...